cmd/mavenBuild.go

```diff
@@ -194,7 +194,7 @@ func loadRemoteRepoCertificates(certificateList []string, client piperhttp.Downl
 		return errors.Wrap(err, "Could not copy existing cacerts into new cacerts location ")
 	}
 
-	if err := fileUtils.Chmod(trustStore, 0644); err != nil {
+	if err := fileUtils.Chmod(trustStore, 0666); err != nil {
```

Comment:
I think in other occurrences we use `755`. Any objections to use that here as well?

Reply:
wanted to only provide read / write permission and not give an execute permission hence had kept it 666

Reply:
:+1